daemon/algod/api/server/v1/handlers/errors.go

```diff
@@ -17,17 +17,21 @@
 package handlers
 
 var (
```

Comment:
The changes here aren't that substantial; I just reorder the errors alphabetically while adding three.